plugins/adr-backend/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@backstage/plugin-adr-backend",
+  "version": "0.1.0",
```

Comment:
Just wanna double check because I've seen this a couple times recently, this actually templates as `0.0.0` and you switched it to `0.1.0` right? 😅  It should be `0.0.0` as we don't release plugins until after the first bump.

Reply:
ah, I may have scaffolded this with an older version of the cli in our internal instance and copied this over to the OSS project, I'll change it to `0.0.0`